cg/models/taxprofiler/taxprofiler.py

```diff
@@ -1,20 +1,19 @@
 from pathlib import Path
 
 from pydantic import Field, BaseModel
-from typing import Optional
+from typing import Union
 
 from cg.constants.sequencing import SequencingPlatform
 from cg.models.nf_analysis import NextflowSampleSheetEntry, PipelineParameters
-from cg.models.analysis import AnalysisModel
 
 
 class TaxprofilerQCMetrics(BaseModel):
     """Taxprofiler QC metrics."""
 
-    after_filtering_total_reads: Optional[float] = None
-    reads_mapped: Optional[float] = None
-    before_filtering_total_reads: Optional[float] = None
-    paired_aligned_none: Optional[float] = None
+    after_filtering_total_reads: Union[float, None]
+    reads_mapped: Union[float, None]
```

Comment:
```suggestion
    reads_mapped: float | None
```
😄

Reply:
Thanks for that, https://github.com/Clinical-Genomics/cg/pull/2701/commits/54a387fa39f68efa131f49183a401ae405ded744